src/styles/block/calendar.scss

```diff
@@ -0,0 +1,59 @@
+.calendar {
+  width: calendarWidth($square-size, $space-between-blocks, $week);
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  gap: $space-between-blocks;
+
+  @for $style-square from 1 through 31 {
+    .day {
+      content: "";
+      position: relative;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      @include square ($square-size);
+      box-sizing: border-box;
+      background-color: #eee;
+      border: 1px solid #000;
+      transition-duration: 0.5s;
+
+      &:hover {
+        cursor: pointer;
+        background-color: pink;
+        transform: translateY(-20px);
```

Comment:
add some your transition properties to the block
default - all, but it can reduce the old browsers performance